src/Components/CreateImageWizard/steps/Registration/SatelliteRegistration.tsx

```diff
@@ -88,12 +87,12 @@ const SatelliteRegistration = () => {
                 ? 'Must be a .PEM/.CER/.CRT file no larger than 512 KB'
                 : validated === 'error'
                 ? stepValidation.errors['certificate']
-                : 'Drag and drop a file or upload one'}
+                : 'Drag and drop a valid certificate file or upload one'}
```

Comment:
Should a valid certificate validate as success in the helper text?

![image](https://github.com/user-attachments/assets/31f4aa9e-984d-4446-abad-fb4d8743131d)